vtex/faststore
# feat: Migrate Newsletter section to app folder

## What's the purpose of this pull request?
Migrate Newsletter section to app folder

## How it works?

![Captura de Tela 2024-08-20 às 16 07 49 (2)](https://github.com/user-attachments/assets/d2bcb9c2-377e-47bb-8641-a4f2fd05d275)




## How to test it?

Run yarn dev
Make sure that all the current pages are working as expected (/, /office and some /<slug>/p);
Make sure /fs-next-update is rendering with the Hero section without errors.
